.github/dependabot.yml

```diff
@@ -24,6 +24,12 @@ updates:
         interval: monthly
       open-pull-requests-limit: 3
       rebase-strategy: disabled
+    - package-ecosystem: gomod
```

Comment:
In the last merge, the dependabot config file is sorted by path, therefore this new module would fall at the end, in the modules block 🙏